src/pages/dev/send-tokens/interchain-tokens.mdx

```diff
@@ -137,62 +147,15 @@ function deployCustomTokenManager(
-
-#### Token Manager
-A contract created by the Interchain Token Service that is capable of locking and releasing tokens, or minting and burning them, depending on its type.
 
-Token Managers must be lock/unlock or mint/burn.
-
-#### Interchain Token
-An ERC-20 contract created by the Interchain Token Service or by yourself that is capable of being transferred between block chains. If the contract implements the `IInterchainToken` interface, it will have knowledge of its own TokenManager and will contain transfer methods directly. If it does not implement this interface, you will need to call functions on the TokenManager to transfer the token.
-
-### Properties
-#### TokenID 
-The ID of the linking between tokens. You can obtain this either
-* Once for each pre-existing ERC20, as a canonical tokenId (`getCanonicalTokenId(address)`)
-* As an `address deployer` with a `bytes32 salt` as a custom tokenId (`getCustomTokenId(address, bytes32)`)
-
-### Operator
-A role for a tokenManager who can set the flowLimit of this manager to limit the influx/outflux of token. 
-
-### Distributor
-A role for tokens that has permission to mint and burn.
-
-
-### Interchain Token Types
-#### Standardized
-A minimal ERC-20, that implements the `IInterchainToken` and `IStandardizedToken` interfaces and can be deployed by the service. 
-
-#### Custom
-A custom ERC-20 that can have any features of functionality you desire, deployed by you. It can optionally implement the `IInterchainToken` interface to transfer methods directly on your token.
-
-#### Canonical
-A canonical token is an interchain token built from an existing token that is available on multiple chains. The token deployed to other chains will be a [Standardized](#standardized) token. There can only be one canonical token for every ERC-20.
+## Other Use cases
+The Interchain Token Service is a powerful tool for moving tokens between blockchains, providing you with all the functionality and power you need to make your token interactive wherever it needs to be. Here are some other possibilities for Interchain Tokens.
 
-#### Mint/Burn
-Incoming token transfers will result in the local minting of new tokens on this blockchain by the Token Manager.
+One of the features of the TokenManager that can be configured is that of the [Operator](../reference/glossary/#operator). The Operator can manage the flow rates of tokens between chains by calling `setFlowLimit` on the Interchain Token Service. 
 
-Outgoing token transfers will result in the local burning of received tokens.
+Additionally, Interchain Tokens can be made executable, which means that they can be sent alongside a standard GMP message. For an example of this, see the example [InterchainTokenExecutable.sol](https://github.com/axelarnetwork/interchain-token-service/blob/example/example-flows-for-stephen/contracts/examples/InterchainTokenExecutable.sol).
```

Comment:
This is OK for now, but you should work with @Foivos to get the examples from that branch merged in.

Reply:
Noted.